lib/WorldState.js

```diff
@@ -200,7 +200,9 @@ module.exports = class WorldState {
      * The state of the Void Trader
      * @type {VoidTrader}
      */
-    [this.voidTrader] = parseArray(deps.VoidTrader, data.VoidTraders, deps);
+    const voidTraderArray = parseArray(deps.VoidTrader, data.VoidTraders, deps);
+    voidTraderArray.sort((a, b) => b.inventory.length - a.inventory.length);
```

Comment:
Could you sort by which has an activation starting first?

Reply:
I was going to do that at first, but I thought this would be a good catch all for future tennocons when the normal baro might not always start before tennocon baro and vice versa. This way tennocon baro will always be the first in the array. I could still change it though.

Reply:
yeah, i don't necessarily want that one first, but it shouldn't really matter long-term, just for the fallback for the deprecated field...

Reply:
Ah alright, will do